packages/ra-core/src/controller/field/ReferenceManyFieldController.spec.tsx

```diff
@@ -1,9 +1,20 @@
 import * as React from 'react';
-import assert from 'assert';
+import expect from 'expect';
 
 import ReferenceManyFieldController from './ReferenceManyFieldController';
 import renderWithRedux from '../../util/renderWithRedux';
 
+// Ignore warnings about act()
+// See https://github.com/testing-library/react-testing-library/issues/281,
+// https://github.com/facebook/react/issues/14769
+const originalError = console.error;
+jest.spyOn(console, 'error').mockImplementation((...args) => {
+    if (/Warning.*not wrapped in act/.test(args[0])) {
+        return;
+    }
+    originalError.call(console, ...args);
+});
```

Comment:
I'm not sure we should do this. We are probably not waiting for side effects to apply where we should.

Reply:
Done!

Reply:
If you have time and noticed those warnings, would you mind trying to add the wait/expect calls needed to make them go away ?

Reply:
Ok. I don't get any warnings now!